File: content/problems/1880-check-if-word-equals-summation-of-two-words/solutions/decimal-fold-word-value/solution.cpp
class Solution {
    int value(const string& word){
        int result=0;
        for(char letter:word)result=result*10+letter-'a';
        return result;
    }
    public: bool isSumEqual(string firstWord,string secondWord,string targetWord){
        return value(firstWord)+value(secondWord)==value(targetWord);
    }
};

File: content/problems/1880-check-if-word-equals-summation-of-two-words/solutions/decimal-fold-word-value/Solution.kt
class Solution {
    private fun value(word:String):Int{
        var result=0;
        for(letter in word)result=result*10+(letter-'a');
        return result
    };
    fun isSumEqual(firstWord:String,secondWord:String,targetWord:String):Boolean=value(firstWord)+value(secondWord)==value(targetWord)
}
